src/libraries/System.Private.CoreLib/src/System/Globalization/Ordinal.cs

```diff
@@ -462,8 +462,7 @@ ref Unsafe.Add(ref valueRef, 1), valueTailLength))
                         // Do a full IgnoreCase equality comparison. SpanHelpers.IndexOf skips comparing the two characters in some cases,
                         // but we don't actually know that the two characters are equal, since we compared with | 0x20. So we just compare
                         // the full string always.
-                        int bitPos = BitOperations.TrailingZeroCount(mask);
-                        nint charPos = (nint)((uint)bitPos / 2); // div by 2 (shr) because we work with 2-byte chars
+                        nint charPos = (nint)(uint.TrailingZeroCount(mask) / sizeof(ushort));
```

Comment:
```suggestion
                        nint charPos = (nint)(uint.TrailingZeroCount(mask) / sizeof(char));
```

@stephentoub This would be slightly clearer, but might not be worth spinning CI again?